src/components/CasesPage/CasesPage.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import cx from 'classnames';
+import { createBemBlockBuilder } from '../../utils';
+import { SubscriptionBanner } from '../SubscriptionBanner/SubscriptionBanner';
+import './CasesPage.scss';
+import downloadIcon from './icons/download.svg';
```

Comment:
For the banner please create `index.js` file to reexport from it
```suggestion
import React from 'react';
import cx from 'classnames';

import { createBemBlockBuilder } from '../../utils';
import { SubscriptionBanner } from '../SubscriptionBanner';

import downloadIcon from './icons/download.svg';

import './CasesPage.scss';
```

Reply:
Fixed